docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -113,7 +117,10 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
         (warehouse, location),
         (warehouse, size),
         (warehouse))`.
- 
+    
+    `GROUP BY GROUPING SETS(GROUPING SETS((warehouse), (warehouse, product)))` is equivalent to 
+    `GROUP BY  GROUPING SETS((warehouse), (warehouse, product))`.
```

Comment:
nit: remove the extra space after GROUP BY

Reply:
Done